packages/fxa-payments-server/Gruntfile.js

```diff
@@ -16,6 +16,7 @@ module.exports = function (grunt) {
           // conflicting IDs for identical terms.
           'src/branding.ftl',
           'src/**/*.ftl',
+          '../fxa-react/components/**/*.ftl',
```

Comment:
Fly-by review / nitpick here, you could probably put these in an array const/var since these paths are referenced 3 times in this file (as well as in the settings gruntfile), so we only have to make the adjustment in one spot if we need to tweak the list in the future.

Reply:
Sounds good, I've updated the gruntfiles for settings and payments to use a const for the source paths.